src/documents/actor.js

```diff
@@ -212,9 +212,10 @@ export default class Actor5e extends Actor {
 
   prepareSkills() {
     const actorData = this.system;
+    const { jackOfAllTrades } = this.flags.a5e;
 
     Object.values(actorData.skills).forEach((skill) => {
-      skill.mod = skill.proficient ? actorData.attributes.prof : 0;
+      skill.mod = skill.proficient ? actorData.attributes.prof : jackOfAllTrades ? Math.ceil(actorData.attributes.prof/2) : 0;
```

Comment:
This should be `floor` not `ceil`. JoAT rounds down:

> Starting at 2nd Level, whenever you make an ability check with a skill or tool you are not proficient with, you add half your proficiency bonus (rounded down).

Reply:
I'm also not a fan of this extended ternary here. Perhaps we should make a new method to calculate the skill modifier for a single skill where this logic can be expanded into a more readable form.